### DISCUSSION TITLE
How does having multiple packageRules work?

### DISCUSSION THREAD

**Discussion by JrobT** [renovatebot/renovate#15688] [Request Help] [answered]
Say I have this config:

```json
{
  "matchPackageNames": ["aws-sdk"],
  "schedule": ["before 9am on the first day of the month"]
},
```

I want to add all packages that start with `@aws-sdk` as well. Could I do:

```json
{
  "matchPackagePrefixes": ["@aws-sdk"],
  "matchPackageNames": ["aws-sdk"],
  "schedule": ["before 9am on the first day of the month"]
},
```

Sorry for the really standard question - I'm just being blind and can't see it in the docs.

**Comment by rarkins** [comment, marked as answer]
Yes you can. The matchPackageX fields are the only ones which are treated as "OR" instead of "AND". All other match fields will be ANDed together.
